examples/one-2-one/src/controllers/supplier-account.controller.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-Supplier
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  del,
+  get,
+  getWhereSchemaFor,
+  param,
+  post,
+  requestBody,
+} from '@loopback/rest';
+import {Account} from '../models';
+import {SupplierRepository} from '../repositories';
+
+export class SupplierAccountController {
+  constructor(
+    @repository(SupplierRepository) protected supplierRepo: SupplierRepository,
+  ) {}
+
+  @post('/suppliers/{id}/accounts', {
```

Comment:
Each supplier has a single account only, please fix the REST URL paths in all controller methods to use `/account` (singular form).